addons/portal/controllers/mail.py

```diff
@@ -49,7 +49,7 @@ def portal_chatter_init(self, thread_model, thread_id, **kwargs):
                 thread, kwargs.get("hash"), kwargs.get("pid"), kwargs.get("token")
             ):
                 partner = portal_partner
-        store.add({"self": Store.one(partner, fields=["active", "avatar_128", "name", "user"])})
+        store.add({"self": Store.one(partner, fields=["active", "avatar_128", "name", "user", "is_public"])})
```

Comment:
Follow alphabetical order, and yes true portal still uses another init, this is a good place to add the field too.